### PR TITLE
Polish app's responsiveness 

### PR DESCRIPTION
**Pull request by PJColombo**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```
Comment (rperez89) [inline .vscode/settings.json]: were we having discrepancies with our formatting?

**Review by rperez89 — COMMENTED** [review]
Hey @PJColombo awesome! thought i noticed that the menu is not hiding, i think we should hide it on small screens wdyt?

Screen Shot 2023-01-17 at 11 36 56 AM

File: .vscode/settings.json (hunk 1 above)
Comment (PJColombo) [reply .vscode/settings.json]: Oops, this shouldn't have been committed. I'll remove it.

**Comment by PJColombo** [comment]
> Hey @PJColombo awesome! thought i noticed that the menu is not hiding, i think we should hide it on small screens wdyt?



Mmmm, that's weird @rperez89 .  That doesn't look like the latest version of this PR :thinking: 

I'm getting this when running the project on dev: 

image

One thing to consider is when changing the dimensions dynamically with the browser the components are not being re-render and re-use the same boolean that controls the drawer's display so when going from `medium` to `small` mode it stays `true`. You can still hide it on `small` view mode tho

**Comment by rperez89** [comment]
@PJColombo  ouch maybe i had something cached, working now. lets just fix the conflict and we can merge

**Pull request merged**